config/install/tripal.tripal_content_terms.trpcultivate_terms.yml

```diff
@@ -66,26 +75,87 @@ vocabularies:
+            # In tripal_chado so must match exactly.
             -   id: 'AGRO:00000379'
                 name: 'data collector'
                 description: 'A person that has a role of collecting data related to the experiment.'
     -   name: 'efo'
         label: 'Experimental Factor Ontology'
-        url: 'https://www.ebi.ac.uk/efo/'
+        url: 'https://www.ebi.ac.uk/ols4/ontologies/efo'
         idSpaces:
             -   name: 'EFO'
                 description: 'Experimental Factor Ontology'
-                urlPrefix: 'http://www.ebi.ac.uk/efo/{db}_{accession}'
+                urlPrefix: 'https://www.ebi.ac.uk/ols4/ontologies/efo/{db}_{accession}'
         terms:
+            # In tripal_chado so must match exactly.
             -   id: 'EFO:0001736'
                 name: 'funder'
                 description: 'A person or organization that supports (sponsors) something through some kind of financial contribution.'
+    -   name: 'PROCO'
+        label: 'Process Chemical Ontology'
+        url: 'https://ontobee.org/ontology/PROCO'
+        idSpaces:
+            -   name: 'PROCO'
+                description: 'Process Chemical Ontology'
+                urlPrefix: 'http://purl.obolibrary.org/obo/{db}_{accession}'
+        terms:
+            -   id: 'PROCO:0000276'
+                name: 'Experimental Design'
+                description: 'The design of an experiment intended to test a hypothesis, and describe or explain empirical data obtained under various experimental conditions'
+    -   name: 'CGO'
+        label: 'Common Greenhouse Ontology'
+        url: 'https://agroportal.lirmm.fr/ontologies/CGO'
+        idSpaces:
+            -   name: 'CGO'
+                description: 'Common Greenhouse Ontology'
+                urlPrefix: 'https://agroportal.lirmm.fr/ontologies/CGO/{db}_{accession}'
+            -   name: 'SOSA'
+                description: 'Semantic Sensor Network Ontology'
+                urlPrefix: 'http://www.w3.org/ns/sosa/{accession}'
+        terms:
+            - id: 'SOSA:FeatureOfInterest'
+              name: 'Feature of Interest'
+              description: 'The thing whose property is being estimated or calculated in the course of an Observation to arrive at a Result or whose property is being manipulated by an Actuator, or which is being sampled or transformed in an act of Sampling.'
+    -   name: 'ENVO'
+        label: 'The Environment Ontology'
+        url: 'https://www.ebi.ac.uk/ols4/ontologies/envo'
+        idSpaces:
+            - name: 'ENVO'
+              description: 'The Environment Ontology'
+              urlPrefix: 'https://www.ebi.ac.uk/ols4/ontologies/envo/{db}_{accession}'
+        terms:
+            - id: 'ENVO:01001406'
+              name: 'Laboratory Facility'
+              description: 'A research facility in which systems of manufactured products control internal conditions and in which scientific or technological research, experiments, and measurement may be performed.'
```

Comment:
This should be help text to the user, not the definition of the term. For example, how should they format it? What should this name include?

Reply:
Has the help text changed into "Include the name or code of the laboratory. Location of the laboratory needs to be included (e.g. City, Province)"